wrappers/wrapper.rb

```diff
@@ -659,6 +659,12 @@ def add_unassigned_internal(unfeasible, vrp, service, reason)
         }
       end
 
+      vrp.routes.each{ |route| route.mission_ids.delete(service.id) }
+      vrp.relations.each{ |relation|
+        relation.linked_ids.delete(service.id)
+        relation.linked_services.delete(service)
+      }
+
```

Comment:
Periodic heuristic already ensures that we do not initialize routes with services that do not exist in current VRP so I will just remove these delete.

Periodic does not do anything on relations because current version of periodic heuristic does not support relations.